test/torchaudio_unittest/functional/batch_consistency_test.py

```diff
@@ -140,53 +146,71 @@ def test_amplitude_to_DB_not_channelwise_clamps(self):
         assert (difference >= 1e-5).any()
 
     def test_contrast(self):
-        waveform = torch.rand(2, 100) - 0.5
-        self.assert_batch_consistencies(
-            F.contrast, waveform, enhancement_amount=80.)
+        torch.random.manual_seed(0)
+        waveforms = torch.rand(self.batch_size, 2, 100) - 0.5
```

Comment:
Could you also take this opportunity to replace these waveform generation with `get_whitenoise`?

Reply:
Sure thing